bin/fmt.sh

```diff
@@ -57,38 +72,45 @@ if [ $GO_FMT_TOOL = "gofmt" ]; then
   fmt_args="-w"
 fi
 
-PKGS=${PKGS:-"."}
-if [[ -z ${GO_FILES} ]];then
-  GO_FILES=$(find "${PKGS}" -type f -name '*.go' ! -name '*.gen.go' ! -name '*.pb.go' ! -name '*mock*.go' | grep -v ./vendor)
-fi
-
 if [ $check = false ]; then
-  for fl in ${GO_FILES}; do
-    if head -20 "$fl" | grep "Code generated by counterfeiter" > /dev/null; then
-       continue
-    fi
-    if head -20 "$fl" | grep "Code generated by go-bindata" > /dev/null; then
-       continue
-    fi
-    $tool $fmt_args "$fl"
-  done
+  echo "formatting the source files"
+  # shellcheck disable=SC2086
+  $tool $fmt_args ${GO_FILES}
+#  for fl in ${GO_FILES}; do
```

Comment:
Remove commented out code?

Reply:
There's a benefit to the commented out code: it bypasses the command-line length restriction on circle. I thought it might be worth saving it if someone decides to re-enable `make fmt` on circle.